### ISSUE TITLE
FastNLP_Bert Accuracy failed for key name bert.model.encoder.embeddings.LayerNorm.weight.grad

### ISSUE THREAD

**Issue by mengfei25** [intel/torch-xpu-ops#490]
### 🐛 Describe the bug

torchbench_amp_fp16_training
xpu  train fastNLP_Bert
W0626 17:49:36.629000 140234806880064 torch/_dynamo/variables/tensor.py:715] [3/0] Graph break from `Tensor.item()`, consider setting:
W0626 17:49:36.629000 140234806880064 torch/_dynamo/variables/tensor.py:715] [3/0]     torch._dynamo.config.capture_scalar_outputs = True
W0626 17:49:36.629000 140234806880064 torch/_dynamo/variables/tensor.py:715] [3/0] or:
W0626 17:49:36.629000 140234806880064 torch/_dynamo/variables/tensor.py:715] [3/0]     env TORCHDYNAMO_CAPTURE_SCALAR_OUTPUTS=1
W0626 17:49:36.629000 140234806880064 torch/_dynamo/variables/tensor.py:715] [3/0] to include these operations in the captured graph.
W0626 17:49:36.629000 140234806880064 torch/_dynamo/variables/tensor.py:715] [3/0]
E0626 17:50:10.260000 140234806880064 torch/_dynamo/utils.py:1478] RMSE (res-fp64): 0.00383, (ref-fp64): 0.00017 and shape=torch.Size([768]). res.dtype: torch.float32, multiplier: 3.000000, tol: 0.001000
E0626 17:50:10.261000 140234806880064 torch/_dynamo/utils.py:1392] Accuracy failed for key name bert.model.encoder.embeddings.LayerNorm.weight.grad
fail_accuracy

### Versions

torch-xpu-ops: https://github.com/intel/torch-xpu-ops/commit/31c400195d63064940242220dc9100322d36bac4
pytorch: 0f81473d7b4a1bf09246410712df22541be7caf3 + PRs: 127277,129120
device: PVC 1100, 803.61, 0.5.1

**Comment by retonym** [comment]
No accuracy issue in latest torch xpu ops
